documentation/release-checklist.md

```diff
@@ -10,6 +10,11 @@
 - [ ]  Create darc channel for `VS {{NEXT_VERSION}}` if it doesn't already exist \
 `darc add-channel --name "VS {{NEXT_VERSION}}"`
 - [ ]  Ping internal "First Responders" Teams channel to get the new channel made available as a promotion target (e.g. dotnet/arcade#12150): {{URL_OF_CHANNEL_PROMOTION_PR}}
+- [ ] Create darc subscriptions for new channel from:
```

Comment:
This is probably quite stable list of subscriptions. But for sure we should probably add how to get the list of subscriptions in the current channel (`darc get-subscriptions --target-repo dotnet/msbuild --channel vs{{THIS_RELEASE_VERSION}}`)

Reply:
I added this because these versions are very stable, so we end up falling behind on updating them, or certain channels that should be receiving updates are not.